docs/pages/example/jump-to.html

```diff
@@ -0,0 +1,95 @@
+<div id='map'></div>
+<script>
+
+var cities = {
+    "type": "FeatureCollection",
+    "features": [
+        {
+            "type": "Feature",
+            "properties": {},
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    100.507,
+                    13.745
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {},
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    98.993,
+                    18.793
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {},
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    99.838,
+                    19.924
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {},
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    102.812,
+                    17.408
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {},
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    100.458,
+                    7.001
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {},
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    100.905,
+                    12.935
+                ]
+            }
+        }
+    ]
+};
+
+mapboxgl.accessToken = 'pk.eyJ1IjoibWFsLXdvb2QiLCJhIjoiY2oyZ2t2em50MDAyMzJ3cnltMDFhb2NzdiJ9.X-D4Wvo5E5QxeP7K_I3O8w';
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/streets-v10',
+    center: [100.507, 13.745],
+    zoom: 9
+});
+
+map.on('load', function() {
+
+    var i = -1;
+    (function jumpToCity () {
```

Comment:
I think this [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) might be a little over-complicated for the purpose of the examples page – begginers to JS might not know what's going on here. 
You could simplify this so that its just a simple loop that only visits each location once, or even simpler just have a button that will jump trigger a jump to a random location or a random location in your list like the `flyTo` example